$OpenBSD: patch-shell_bootstrap-ocaml_sh,v 1.1 2019/03/12 05:36:06 chrisz Exp $

use clang instead of gcc when bootstrapping OCaml,
not necessary at the moment, because we use the installed compiler

Index: shell/bootstrap-ocaml.sh
--- shell/bootstrap-ocaml.sh.orig
+++ shell/bootstrap-ocaml.sh
@@ -130,7 +130,11 @@ if [ -n "$1" -a -n "${COMSPEC}" -a -x "${COMSPEC}" ] ;
 else
   PREFIX=`cd .. ; pwd`/ocaml
   if [ ${GEN_CONFIG_ONLY} -eq 0 ] ; then
-    ./configure -prefix "${PREFIX}"
+    ./configure -prefix "${PREFIX}" \
+		-aspp '${CC} -c' \
+		-cc ${CC} \
+		-x11include ${X11BASE}/include \
+		-x11lib ${X11BASE}/lib
     ${MAKE:-make} world opt.opt
     ${MAKE:-make} install
   fi
